### ISSUE TITLE
 metadata using csvy

### ISSUE THREAD

**Issue by jenmunoz** [BIOL548O/Discussion#26]
I am wondering if when the reviewers run the Rscript for the metadata .csvy, it will rewrite the file and delete all the info included in the metadata manually. That is not ideal!

**Comment by aammd** [comment]
@jenmunoz I completely agree that its not ideal! I think that a good solution is to somehow separate the file with the metadata from the one that your script produces. Here are some options:
- have  your script write a file like `mydata_tidy.csv` and then create a metadata file called `mydata_with_metadata.csvy` or something.
- You could write a CSV, then create your YAML metadata by hand and save the new file as `.csvy` (then they can have the same name, but different file extension)
- Write your YAML by hand and keep it in a separate `metadata.yml` file.

But whatever happens, there is _no risk_ of your reviewers accidentally modifying your work. I've made sure that they do not have permission to do so!
